src/collection.js

```diff
@@ -79,13 +79,18 @@ export default class Collection {
    *
    * @param  {Object} [options={}]      The options object.
    * @param  {Object} [options.headers] The headers object option.
+   * @param  {String} [options.since]   The since ETag option.
    * @return {Promise<Number, Error>}
    */
   getTotalRecords(options={}) {
     const { headers } = this._collOptions(options);
```

Comment:
So that means `since` is passed to `_collOptions` and merged into the instance options. I don't think that it's a problem per se, but explicitely removing it may be nicer:

```js
getTotalRecords(options={}) {
  const { since, ...otherOptions } = options;
  const { headers } = this._collOptions(otherOptions);
  let path = endpoint("record", this.bucket.name, this.name);
  if (since) {
    path += "?since="+options.since;
  }
```